examples/chain/44_DriftSolver/1_PlaceOrder/example.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	astromeshtypes "github.com/FluxNFTLabs/sdk-go/chain/modules/astromesh/types"
+	strategytypes "github.com/FluxNFTLabs/sdk-go/chain/modules/strategy/types"
+	chaintypes "github.com/FluxNFTLabs/sdk-go/chain/types"
+	chainclient "github.com/FluxNFTLabs/sdk-go/client/chain"
+	"github.com/FluxNFTLabs/sdk-go/client/common"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/gagliardetto/solana-go"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+func main() {
+	network := common.LoadNetwork("local", "")
+	kr, err := keyring.New(
+		"fluxd",
+		"file",
+		os.Getenv("HOME")+"/.fluxd",
+		strings.NewReader("12345678"),
+		chainclient.GetCryptoCodec(),
+	)
+	if err != nil {
+		panic(err)
+	}
+
+	// init grpc connection
+	cc, err := grpc.Dial(network.ChainGrpcEndpoint, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	if err != nil {
+		panic(err)
+	}
+
+	// init client ctx
+	clientCtx, senderAddress, err := chaintypes.NewClientContext(
+		network.ChainId,
+		"user1",
+		kr,
+	)
+	if err != nil {
+		panic(err)
+	}
+	clientCtx = clientCtx.WithGRPCClient(cc)
+
+	// init chain client
+	chainClient, err := chainclient.NewChainClient(
+		clientCtx,
+		common.OptionGasPrices("500000000lux"),
+	)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
let's do `panic` here so it crashes at the line 

`fmt.Println(err)` => `panic(err)`